contracts/MatchPayouts.sol

```diff
@@ -75,16 +75,16 @@ contract MatchPayouts {
   /**
    * @param _owner Address of contract owner
    * @param _funder Address of funder
-   * @param _dai DAI address
+   * @param _token ERC20 token address
    */
   constructor(
     address _owner,
     address _funder,
-    IERC20 _dai
+    IERC20 _token
   ) {
     owner = _owner;
     funder = _funder;
-    dai = _dai;
+    token = _token;
```

Comment:
note that this introduces a possible security hole.   the deployer should make sure that the contract being input into  the smart contract does not contain a [reentrancy attack](https://quantstamp.com/blog/what-is-a-re-entrancy-attack)

Reply:
So if you check the doc:

```
Claiming Funds: Some grants use contract wallets and it may not be easy for them to call a method allowing them to claim funds. As a result, the `claimMatchPayout` method allows anyone to withdraw on behalf of a grant, and transfers the funds to that grant's receiving address. Additionally, because there is no `msg.sender` usage, and because all match payouts are in a single ERC20 token, there is no reentrancy risk to worry about here.
```